pyiron/vasp/base.py

```diff
@@ -1193,16 +1193,52 @@ def append_wave_function(self, job_specifier=None, path=None):
         else:
             self.restart_file_list.append(posixpath.join(path, "WAVECAR"))
 
-    def set_spin_constraint(self, direction=False, norm=False):
+    def set_rwigs(self, rwigs_dict):
         """
-        Setting thr spin constraints
+        Sets the radii of Wigner-Seitz cell. (RWIGS tag)
 
         Args:
-            direction:
-            norm:
+            rwigs_dict (dict): Dictionary of species type and corresponding radius.
 
         Returns:
+        """
+        species_keys = self.structure.get_number_species_atoms().keys()
+        rwigs_keys = rwigs_dict.keys()
+        for i in species_keys:
+            if i not in list(rwigs_keys):
+                raise ValueError("'" + i + "' is not in rwigs_dict!")
+
+        rwigs = [rwigs_dict[i] for i in species_keys]
+        self.input.incar['RWIGS'] = ' '.join(map(str, rwigs))
+
+    def get_rwigs(self):
+        """
+        Gets the radii of Wigner-Seitz cell. (RWIGS tag)
+
+        Returns:
+            rwigs_dict (dict): dictionary of radii
```

Comment:
`dict: dictionary of radii` instead of `rwigs_dict (dict): dictionary of radii`. The docstrings worn't be able to render properly otherwise!

Reply:
Thanks! Implemented.